test/parallel/test-os.js

```diff
@@ -191,3 +181,20 @@ is.string(pwd.username);
 assert.ok(pwd.homedir.includes(path.sep));
 assert.strictEqual(pwd.username, pwdBuf.username.toString('utf8'));
 assert.strictEqual(pwd.homedir, pwdBuf.homedir.toString('utf8'));
+
+// Test that the Symbol.toPrimitive functions work correctly
+/* eslint-disable no-restricted-properties */
+[
+
+  [assert.equal, `${os.freemem}`, os.freemem()],
```

Comment:
Sorry, I know what you meant, just wasn't being very clear. I'm using `assert.equal()` intentionally because the return value of the methods themselves are numbers and I want to ensure that the conversion to string after the `Symbol.toPrimitive` call returns an equal (but not strict equal) result. Either way I think gets the job done but to answer the question: yes, I prefer it this way :-)